single-node-k8s.sh

```diff
@@ -110,6 +110,7 @@ function deploy_rook()
 {
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh deploy-rook
     ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-block-pool
+    ${GOPATH}/src/github.com/ceph/ceph-csi/scripts/minikube.sh create-block-ec-pool
```

Comment:
This means that the change in `rook.sh` in the `devel` branch needs to be backported to all active `release-*` branches.

Reply:
Yes correct at least to 3.4 branch

Reply:
Should we ask mergify to backport https://github.com/ceph/ceph-csi/pull/2654 to release v3.4?

Reply:
yes, add the required backport labels

Reply:
Done